<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinishDialog</class>
 <widget class="QDialog" name="FinishDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="box1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="box2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>301</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>231</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>button</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
